tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -69,6 +69,8 @@ def get_loader(test_paths, product, debug=None, run_info_extras=None, chunker_kw
     include = kwargs["include"]
     if test_groups:
         include = testloader.update_include_for_groups(test_groups, include)
+    elif kwargs["include_file"] is not None:
+        include = testloader.update_include_from_file(kwargs["include_file"])
```

Comment:
So the behavior here is now that:

1. `--include` will be overridden by either `--test-groups` **or** `--include-file`
2. `--include-file` will be overridden by `--test-groups`

Am I correct, and is that what you expected?

Reply:
Thanks! Actually I want to hear your suggestions. I am thinking people should not use "--include" and "--include-file" together, and they should not use "--include-file" and "--test-groups" together.

Reply:
I would much prefer we just make `--include-file` append the lines from the file to `kwargs["include"]` so that it is exactly equivalent to passing the same parameters on the command line i.e.
```
include = kwargs["include"]
if kwargs["include_file"]:
    with open(kwargs["include_file"]) as f:
        for line in f:
            line = line.strip()
            # Allow whole-line comments; fragments mean we can't have partial line #-based comments
            if line.startswith("#"):
                continue
            include.append(line)
    if test_groups:
        include = testloader.update_include_for_groups(test_groups, include)
```

Although of course the middle part can be factored out into a function.

Reply:
I agree with jgraham's suggestion here, making `--include` and `--include-file` cumulative seems sensible to me.

Reply:
Thanks folks. I updated the PR again based on the comments. Please help review again.